xarray/core/rolling.py

```diff
@@ -917,6 +953,28 @@ def wrapped_func(self, **kwargs):
 
         return wrapped_func
 
+    def reduce(self, func: Callable, **kwargs):
+        """Reduce the items in this group by applying `func` along some
+        dimension(s).
+
+        Parameters
+        ----------
+        func : callable
+            Function which can be called in the form  `func(x, axis, **kwargs)`
+            to return the result of collapsing an np.ndarray over the coarsening
+            dimensions.  It must be possible to  provide the `axis` argument with
+            a tuple of integers.
+        **kwargs : dict
+            Additional keyword arguments passed on to `func.
```

Comment:
```suggestion
            Additional keyword arguments passed on to `func`.
```

Reply:
Perfect, thanks!  Was wondering why the doc build failed...